test/feature/features/coordinator.feature

```diff
@@ -129,21 +128,6 @@ Feature: Coordinator test
     UNREGISTER ROUTER r1;
     REGISTER ROUTER r1 ADDRESS regress_router::7000
     """
-    When I run SQL on host "router-admin"
```

Comment:
Is there any ways to see list of relation and columns?

Reply:
There aren't now, I suggest adding more output to `SHOW distributions` command in separate PR